velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,18 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the (signed) number of days since unix epoch (1970-01-01).
+/// Returns UserError status if the date is invalid.
+/// @param year Year, can be negative e.g: 1996, -2000
+/// @param month Month of year, e.g: 7
```

Comment:
Would you clarify whether it is 0- or 1-based? Are valid values [0, 11] or [1, 12]?

Reply:
I changed comments. Thanks.